### ISSUE TITLE
Update assets

### ISSUE THREAD

**Issue by wooorm** [get-alex/alex#261]
Due to accessibility problems (GH-258), we should use a slightly darker purple, so that the white-on-purple is more legible.

We have a couple of media assets in [`media/`](https://github.com/get-alex/alex/blob/master/media/header.svg), and on the website branch in [`src/`](https://github.com/get-alex/alex/tree/website/src).

These media and website assets are also different, because they were created by different people.

Another problem is that the favicon is generated from a larger photo, making it super blurry in the 16x16 size.

### Design spec

- Background colour: `#6f42c1`
- Foreground colour: `#ffffff`
- Font: Preferably San Francisco Compact Display Regular
- Either uppercase or lowercase, not capitalised, which ever looks better
- Word (`alex`) when there’s space, or initial (`a`) when there isn’t
- Equals to sign under the word or initial

### Assets

- `alex-header.png` is used for [`og:image`](https://developers.facebook.com/docs/sharing/best-practices#images) and [`twitter:image`](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started)
- Android / apple touch (one 180x180?) / mstile icons (150x150) (PNG)
- Safari pinned tab (SVG)
- Actual favicons
- An icon for our GH org: https://github.com/get-alex

--- 

/cc @danoszz, could you help with this?

**Comment by danoszz** [comment]
@wooorm yes of course. I will make a PR when it is ready. Thanks for the extensive design spec.

**Comment by wooorm** [comment]
OMGosh this is beau-ti-ful! 😍!

My only question is that I’m guessing that the GH org logo may also be a bit small, e.g., looking at my profile under organizations, it seems to be 35x35.

So maybe either use less padding on it, or alternatively, we could also this nice favicon instead?
